### DISCUSSION TITLE
20240423-fe-part/

### DISCUSSION THREAD

**Discussion by giscus** [inflearn/tech-blog-comment#56] [Comments]
# 20240423-fe-part/

https://tech.inflab.com/20240423-fe-part/

**Comment by PyeongBee** [comment]
제목에 이끌려 들어왔지만, 이 글의 끝까지 집중하고 읽게 되는 저를 보며 이 글의 주제와 다른 생각이 들었네요.

'글을 정말 흡입력 있게, 의식의 흐름대로 자연스럽게 잘 작성하신다...'

이 글의 '프론트엔드 개발자로서 얻을 수 있는 것들' 부터 '마무리'까지.
'아, 다른 사람들과의 차별점을 이렇게 고민을 해봐야겠다.'
'블로그에 글을 포스팅할 때에도 평소 소홀히 하던 글의 인트로와 아웃트로를 신경써야겠구나.'

좋은 글 감사합니다!
이걸 전달하고자 작성하신 글 같지는 않지만 깨달음을 얻고 가네요...! 🤭

**Comment by Young2un** [comment]
기업 입장에선 대체 가능한 인재 그리고 스스로는 대체 불가능한 인재가 되어야 매력적이겠구뇽!
